### DISCUSSION TITLE
How to fix this color issue on path.filename_first?

### DISCUSSION THREAD

**Discussion by serranomorante** [ibhagwan/fzf-lua#1175] [Q&A] [answered]
Was trying the path.filename_first feature but _path_ part appears in black. Is there anything I can do to fix this on `files` and `buffers`?

Thanks!

![Screenshot_20240504_224306](https://github.com/ibhagwan/fzf-lua/assets/43413954/9f29baaa-5f11-43ab-a331-35daf32cab01)

**Comment by ibhagwan** [comment]
Since I just released this feature I hardcoded this part to the terminal grey which is more black in your term colors.

I intend to make a highlight group for this linked by default to neovim’s `Comment` hl group, will be probably get to it over the weekend.

**Reply by serranomorante** [reply]
Thank you for the clarification! Until then I will change the color manually in the source code and use it locally 👍

**Comment by ibhagwan** [comment, marked as answer]
https://github.com/ibhagwan/fzf-lua/commit/e32f3dfb543c69fe3cc6a85b763f66b7ff98294e

The above commit defines a new highlight group `FzfLuaDirPart` (linked to `Comment` by default):
> Can be linked to anything
```lua
:hi! link FzfLuaDirPart IncSearch
```
> defined in `setup`
```lua
require("fzf-lua").setup({
  hls = { dir_part = "IncSearch" }
})
```

> Or sent directly in a call
```lua
:FzfLua live_grep formatter=path.filename_first hls.dir_part=ErrorMsg
```

![image](https://github.com/ibhagwan/fzf-lua/assets/59988195/af36433b-586b-46cf-a322-420717069c5d)


FYI, for an easy way to set this for all pickers that contain path entries, use global picker defaults in `setup.defaults`:
```lua
require("fzf-lua").setup({
  defaults = { formatter = "path.filename_first" },
  -- rest of your setup
})
```

**Reply by serranomorante** [reply]
Thank you so muuuuchhh!!! Now the default `Comment` highlight works perfectly. :heart:
